Assets/Packages/com.studio23.ss2.achievementsystem/Runtime/Data/AchievementData.cs

```diff
@@ -0,0 +1,30 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Studio23.SS2.AchievementSystem.Data
+{
+	[Serializable]
+	[CreateAssetMenu(fileName = "New AchievementData", menuName = "Studio-23/Achievement System/New Achievement")]
+	public class AchievementData : ScriptableObject
+	{
+		public string AchievementName;
+		public string AchievementDescription;
+        public Texture2D LockedIcon;
```

Comment:
Please fix inconsistent code formatting

Reply:
This is probably a github issue. It doesn't exist in code. Fixed